cpp/ql/test/library-tests/scopes/parents/parents.expected

```diff
@@ -14,7 +17,11 @@
 | 1 | parents.cpp:4:10:4:10 | f | parents.cpp:4:19:13:5 | { ... } |
 | 1 | parents.cpp:4:19:13:5 | { ... } | parents.cpp:5:11:5:11 | j |
 | 1 | parents.cpp:4:19:13:5 | { ... } | parents.cpp:6:11:10:7 | { ... } |
+| 1 | parents.cpp:4:19:13:5 | { ... } | parents.cpp:11:18:11:18 | e |
```

Comment:
It feels like the parent of `e` should be something associated with the catch block on line 11, rather than the entire function.  I can see that this wouldn't fit the current definition of what `getParentScope` can return.  And what you have is probably good enough for what `getParentScope` is used for.

Reply:
Yeah, agreed. I struggled a bit with that one initially. What I do now seems to be most in line with all the other case that are there, and what we want when looking for shadowing.